app/mailers/startup_mailer.rb

```diff
@@ -2,8 +2,11 @@
 class StartupMailer < SchoolMailer
   def feedback_as_email(startup_feedback)
     @startup_feedback = startup_feedback
+
+    # it's intended to send only to one recipient
     send_to = startup_feedback.timeline_event.founders.map { |e| "#{e.fullname} <#{e.email}>" }
     @school = startup_feedback.startup.school
+    @student_fullname = startup_feedback.timeline_event.founders.first.fullname
```

Comment:
Take a closer look to `founders.first`

Reply:
@prashanth726 do `timeline_event`s only have one founder/student or can they have multiple?